templates/admin/users_inactive.html.twig

```diff
@@ -0,0 +1,77 @@
+{% extends 'base.html.twig' %}
+
+{%- block title -%}
+    {{- 'users'|trans }} - {{ parent() -}}
+{%- endblock -%}
+
+{% block mainClass %}page-admin-users{% endblock %}
+
+{% block header_nav %}
+{% endblock %}
+
+{% block sidebar_top %}
+{% endblock %}
+
+{% block body %}
+    {% include 'admin/_options.html.twig' %}
+
+    <div class="pills">
+        <menu>
+            <li>
+                <a href="{{ path('admin_users_active') }}"
+                   class="{{ html_classes({'active': is_route_name('admin_users_active')}) }}">
+                    {{ 'admin_users_active'|trans }}
+                </a>
+            </li>
+            <li>
+                <a href="{{ path('admin_users_inactive') }}"
+                   class="{{ html_classes({'active': is_route_name('admin_users_inactive')}) }}">
+                    {{ 'admin_users_inactive'|trans }}
+                </a>
+            </li>
+            <li>
+                <a href="{{ path('admin_users_suspended') }}"
+                   class="{{ html_classes({'active': is_route_name('admin_users_suspended')}) }}">
+                    {{ 'admin_users_suspended'|trans }}
+                </a>
+            </li>
+            <li>
+                <a href="{{ path('admin_users_banned') }}"
+                   class="{{ html_classes({'active': is_route_name('admin_users_banned')}) }}">
+                    {{ 'admin_users_banned'|trans }}
+                </a>
+            </li>
+        </menu>
+    </div>
```

Comment:
you could put this in its own file and just include it

Reply:
makes sense!